test/sslapitest.c

```diff
@@ -2027,6 +2053,327 @@ static int test_tlsext_status_type(void)
 
     return testresult;
 }
+
+# ifndef OSSL_NO_USABLE_TLS1_3
+static int ocsp_server_cb_multi(SSL *s, void *arg)
+{
+    int *argi = (int *)arg;
+    const SSL_CONNECTION *sc = SSL_CONNECTION_FROM_CONST_SSL(s);
+    ASN1_TIME *thisupd = NULL, *nextupd = NULL;
+    X509 *ssl_cert = NULL;
+    X509 *issuer = NULL;
+    int i, num = 0;
+    STACK_OF(X509) *server_certs = NULL;
+    OCSP_RESPONSE *ocsp_resp;
+    STACK_OF(OCSP_RESPONSE) *sk_resp = NULL;
+    OCSP_CERTID *cert_id = NULL;
+    OCSP_BASICRESP *bs = NULL;
+    char *signer_key_file = test_mk_file_path(certsdir, "subinterCA.key");
+    char *signer_cert_file = test_mk_file_path(certsdir, "subinterCA.pem");
```

Comment:
Again, for correct signers of the OCSP responses,
you cannot use `subinterCA` for all of them!
For the 2nd one, must use `interCA`, and for the 3rd one `rootCA`.